client/src/components/AddClimbedBoulderForm.spec.jsx

```diff
@@ -21,7 +31,12 @@ describe('AddClimbedBoulderForm', () => {
 
   it('submits form data if none of the input fields is filled out', () => {
     const handleAdd = jest.fn();
-    render(<AddClimbedBoulderForm saveClimbedBoulderToDatabase={handleAdd} />);
+    render(
+      <AddClimbedBoulderForm
+        formPrefilledClimbedBoulder={climbedBoulder}
+        saveClimbedBoulderToDatabase={handleAdd}
+      />
+    );
```

Comment:
I don't really understand why you would like to test whether `any kind of data` is submitted if none of the input fields were filled out. I think it would make more sense to test that you receive the same prefilled data you hand over to you `AddClimedBoulderForm` as prefilled data, what do you think?

Reply:
You are right. I have adjusted the Form tests now so that I test it with an empty object and with an object containing data. And that made me think that I could adjust the server response to be an empty object as well. So I updated the code in api.js too. Please have a look.